doc/containers.md

```diff
@@ -160,7 +160,7 @@ ID (database)   | Name                              | Description
 5               | [usb](#type-usb)                  | USB device
 6               | [gpu](#type-gpu)                  | GPU device
 7               | [infiniband](#type-infiniband)    | Infiniband device
-8               | [proxy] (#type-proxy)             | Proxy device
+8               | [proxy](#type-proxy)             | Proxy device
```

Comment:
Table looks misaligned now :) You want one more space in the name column now so that the description is properly aligned.

Reply:
...yeah